app/operations/fdsh/jobs/generate_transmittable_ssa_payload.rb

```diff
@@ -0,0 +1,139 @@
+# frozen_string_literal: true
+
+module Fdsh
+  module Jobs
+    # create job operation that takes params of key (required), started_at(required), publish_on(required), payload (required)
+    class GenerateTransmittableSsaPayload
+      include Dry::Monads[:result, :do, :try]
+
+      def call(params)
+        values = yield validate_params(params)
+        _job = yield create_job(values)
+        @transmission = yield create_transmission(values)
```

Comment:
Can this be refactored to use local variables instead of instance variables (@ job, @ transmission, and @ transaction)?

Reply:
That was the initial idea but had to change it to use instance variables to avoid passing around to each method and it was much cleaner this way.